### DISCUSSION TITLE
Use without Wagtail

### DISCUSSION THREAD

**Discussion by bastianhilton** [torchbox/wagtail-grapple#191] [Q&A]
I love this idea and would love to know if this can be used without Wagtail. For instance, i built a custom cms on top of Django-cms and Oscar commerce and like mentioned in your documentation on github, its difficult to add graphql to an exisiting project especially a large one without errors.

**Comment by zerolab** [comment]
Hey @bastianhilton,

at the moment this is very much Wagtail-focussed and I do not foresee us branching out soon mainly because Wagtail is what we use day to day. 

You could take inspiration from the implementation for your custom cms. Or, perhaps work with the django-cms community on a package specifically designed for django-cms.
